app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -421,6 +421,8 @@
     <string name="third_party_licenses">使用的第三方库</string>
     <string name="event_owner_detail_organiser">组织者</string>
     <string name="past_event_error_message">Past events cannot be published.</string>
+    <string name="warning_preference_key">experimental_warning_shown</string>
+    <string name="dialog_message">This mobile app is still in its prototype stage with many features being experimental. We currently recommend using this app only for testing.</string>
```

Comment:
This is not chinese, why is it in Chinese strings?

Reply:
Sir if I am not storing the message string in Chinese Strings.It is failing the Build.

Reply:
what I can do is translate the string to Chinese and then add to the Chinese Strings.

Reply:
Disable that lint